service/conf/routes

```diff
@@ -5,6 +5,7 @@ OPTIONS    /*all                        @controllers.LearnerController.preflight
 GET      /v1/user/courses/list/:uid	        @controllers.LearnerController.getEnrolledCourses(uid:String)
 POST     /v1/user/courses/enroll	    @controllers.LearnerController.enrollCourse()
 POST     /v1/course/enrol	    		@controllers.LearnerController.enrollCourse()
+POST     /V1/course/unenroll            @controllers.courseenrollment.CourseEnrollmentController.unenrollCourse()
```

Comment:
V1 -> v1

Reply:
Change links for getEnrolledCourses and enrollCourse from LearnerController to CourseEnrollmentController.